Android/app/build.gradle

```diff
@@ -129,7 +129,7 @@ dependencies {
     implementation("com.google.android.gms:play-services-code-scanner:16.0.0")
 
     // Ditto
-    implementation("live.ditto:ditto:4.4.4")
+    implementation("live.ditto:ditto:4.5.4")
 //    implementation project(":dittoinfomodule")
```

Comment:
Is it useful to leave in the commented line?

Reply:
Not sure what that was for. My assumption is it was something Fuad was using.